tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import pytest
+import sqlalchemy
+
+TEST_SETTINGS = {"database_url": os.environ.get("TEST_DATABASE_URL", "sqlite://")}
```

Comment:
We can change this back to postgress when we merge: https://github.com/hypothesis/data-tasks/pull/3